app/code/Magento/InventoryInStorePickupFrontend/view/frontend/web/js/view/store-selector.js

```diff
@@ -163,7 +163,13 @@ define([
          * @returns {*}
          */
         updateNearbyLocations: function (address) {
-            var self = this;
+            var self = this,
+                skus = [],
+                items = quote.getItems();
+
+            _.each(items, function (item) {
```

Comment:
Will it work with dynamic skus? For example configurable product with custom options?

Reply:
Fixed